marlowe-playground-client/static/css/panels.scss

```diff
@@ -208,6 +214,55 @@ button.minus-btn:hover {
   margin-bottom: 0.5rem;
 }
 
+.action-group {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.action-group-explanation {
+  margin-left: 2rem;
```

Comment:
Try not to use absolute measures and use the `var(--medium-margin);` or `var(--small-margin);` instead.